1.8/faq/database-errors.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: page
+title: "Fixing Database errors"
+categories: [faq]
+---
+
+MyBB set-up is made to be as simple as possible, however sadly sometimes there are a few issues that can occur. Including, it seems, database connection issues.
+With this FAQ tutorial, we will go over how to fix common database issues.
+
+Note: Please find the set-up installation guide for MyBB here: http://docs.mybb.com/1.8/install/

```

Comment:
It seems out of place for me - a reference to install guide suddenly in a database errors guide.